resource_customizations/argoproj.io/Rollout/actions/action_test.yaml

```diff
@@ -47,6 +53,8 @@ discoveryTests:
       disabled: true
     - name: promote-full
       disabled: false
+    - name: pause
+      disabled: false
```

Comment:
Similar here, has pause condition seems to mean that it's already paused.

Reply:
There are two types of "pause" one done using "paused" in spec and one using "status". The pause in the status can be temporary and as a part of the rollout. If the rollout is paused temporarily as a part of the rollout, the user would still want to pause the rollout permanently.